totp/totp.py

```diff
@@ -26,10 +41,10 @@ def generate_code(secret: str,
 
     if seconds_since_the_epoch is None:
         seconds_since_the_epoch = int(time.time())
+
     seconds_since_the_epoch = int(seconds_since_the_epoch // time_window)
 
     code = str(seconds_since_the_epoch)[-4:]
```

Comment:
code is not cryptographically strong, is the same for each user and easily guessable